### PR TITLE
[14.0][MIG] stock_picking_line_sequence

### PR DESCRIPTION
**Pull request by cjallais**
Migration from v12.0 to 14.0 of stock_picking_line_sequence

### CODE REVIEW

**Comment by kittiu** [comment]
This module is required by https://github.com/OCA/purchase-workflow/pull/1060
Similar to that, can you,
* Set the name of this PR for easy search.
* Follow the guideline, and make 2 commits to ease reviewer.

Thank you :)

File: stock_picking_line_sequence/models/stock.py [hunk 1]
```diff
@@ -31,10 +31,37 @@ def create(self, values):
         return move
 
 
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    def _get_aggregated_product_quantities(self, **kwargs):
```
Comment (rousseldenis) [inline stock_picking_line_sequence/models/stock.py:37]: Could you add tests for this part ?

Comment (cjallais) [reply stock_picking_line_sequence/models/stock.py:37]: Done. See last function in test file.

**Review by rousseldenis — APPROVED** [review]

**Comment by bodedra** [comment]
I have tested it on runbot with Sale module.

I have created a order with 4 order lines and confirm it. Verify DO, sequence for all stock moves are 9,999

Screenshot from 2021-02-05 10-42-52

Is this expected?

**Review by HaraldPanten — APPROVED** [review]

**Comment by cjallais** [comment]
Hello,

Is there someone here that can merge this? It's ready to merge. Need it to be able to merge a dependant module in another repository.
Thanks
Cécile Jallais

**Comment by rousseldenis** [comment]
@dreispt a question was on suspend...

**Comment by dreispt** [comment]
Hmm, last comment is a request to merge.
Can you open an Issue with the pending question please?

**Pull request merged**